lib/rspec/instafail.rb

```diff
@@ -4,14 +4,14 @@ module RSpec
   lib = File.expand_path(File.dirname(File.dirname(__FILE__)))
   $LOAD_PATH << lib unless $LOAD_PATH.include?(lib)
 
-  if defined?(::RSpec::Core) && ::RSpec::Core::Version::STRING >= '3.0.0'
-    require "rspec/instafail/rspec_3"
-  else
-    begin
+  if defined?(::RSpec::Core)
+    if ::RSpec::Core::Version::STRING >= '3.0.0'
+      require "rspec/instafail/rspec_3"
+    else
       require "rspec/instafail/rspec_2"
-    rescue LoadError # try rspec 1
-      require "rspec/instafail/rspec_1"
     end
+  elsif defined?(::Spec::VERSION)
```

Comment:
I'd rather leave the rspec1 code alone, if it worked before it will work now ... or did you have issues with rspec 1 ?